op-node/p2p/pings_test.go

```diff
@@ -0,0 +1,79 @@
+package p2p
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/slog"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+
+	"github.com/ethereum-optimism/optimism/op-service/clock"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+)
+
+func TestPingService(t *testing.T) {
```

Comment:
It is probably reasonable to say that ping service *should* not affect network performance or stability, and I don't think we need to go crazy writing integration tests to that end. But what can we do after this feature is merged to confirm that it's performing as expected? We should turn this on on some dev/test nets I suppose?

Reply:
Yes, it's low risk enough, and there's a flag to disable it. We can roll it out to devnet to get some confidence in this feature.